lib/node/runner/checker_ballot_transaction.go

```diff
@@ -346,12 +346,30 @@ func ValidateOp(st *storage.LevelDBBackend, source *block.BlockAccount, op opera
 		if bo.Type == operation.TypeUnfreezingRequest {
 			return errors.UnfreezingRequestAlreadyReceived
 		}
+	case operation.TypeInflationPF:
+		var ok bool
+		var casted operation.InflationPF
+		if casted, ok = op.B.(operation.InflationPF); !ok {
+			return errors.TypeOperationBodyNotMatched
+		}
+		exists, err := block.ExistsBlockOperation(st, casted.VotingResult)
+		if err != nil || !exists {
+			return errors.InflationPFResultMissed
+		}
+		commonAccount, err := GetCommonAccount(st)
```

Comment:
Getting common account is already checked when node runner starts and `runner.NodeRunner.CommonAccountAddress` has common account address. We can use it.